charts/neo4jv5-internal/values.yaml

```diff
@@ -72,10 +72,6 @@ neo4j:
   podSpec:
     initContainers:
       - name: git-sync
-        securityContext:
-          allowPrivilegeEscalation: false
-          capabilities:
-            drop: ['ALL']
```

Comment:
Because when looking at manifests after the deployment I cannot see this security added, 

And looking on official documentation about securityContext on initContainers I can't find anything relevant

If you agree with this PR then I will remove this one: https://github.com/Linkurious/docker-neo4j/pull/128

And there will be a related PR on k8s repo to add an exclude for policy checking this